lib/jekyll/rp_logs/rp_logline.rb

```diff
@@ -145,7 +145,13 @@ def mergeable_with?(next_line)
       end
 
       def merge!(next_line)
-        @contents += "#{space_between_lines}#{next_line.contents}"
+        if /\.\.\.\w*$/.match(@contents) && /^\w*\.\.\./.match(next_line.contents)
```

Comment:
in case the client introduces a trailing space after the "... " and a leading space infront of the next line " ..."The other way doesn't matter, since we're looking at the end of line on the first and the begining of the line on the second.